scripts/run_app.sh

```diff
@@ -10,8 +10,8 @@ npm run frontend-build:production
 echo ""
 
 # Use default environment vars for localhost if not already set
-export DM_DATA_API_URL=${DM_DATA_API_URL:=http://localhost:5000}
-export DM_DATA_API_AUTH_TOKEN=${DM_DATA_API_AUTH_TOKEN:=myToken}
+export DM_DATA_API_URL=${DM_API_URL:=http://localhost:5000}
```

Comment:
`DM_DATA_API_URL` is the name of the new variable that all apps were switched to back in April. Are we going back to the old one or was this an accidental change?